app/views/static_pages/home.html.erb

```diff
@@ -0,0 +1,25 @@
+<h4 class="text-center my-4"><%= t ".newest_story" %></h4>
+<div class="row">
+  <% @stories.limit(6).each do |story| %>
```

Comment:
Where is your config men 😂😂😂

Reply:
:v chết, em lại quên rồi, để em thêm vào ạ.